src/lxc/console.c

```diff
@@ -701,10 +701,53 @@ static int lxc_console_peer_default(struct lxc_console *console)
 	return ret;
 }
 
+int lxc_console_write_ringbuffer(struct lxc_console *console)
+{
+	char *r_addr;
+	ssize_t ret;
+	uint64_t used;
+	struct lxc_ringbuf *buf = &console->ringbuf;
+
+	/* There's not log file where we can dump the ringbuffer to. */
+	if (console->log_fd < 0)
+		return 0;
+
+	/* The log file is simply appended to. */
+	if (console->log_size == 0)
+		return 0;
+
+	/* The log file is rotated. */
```

Comment:
Sorry, why do you not dump the final contents on exit if the user does not want log files rotated?

Reply:
You're right, we should dump it.